packages/docs/pages/select.tsx

```diff
@@ -1,399 +1,499 @@
-              {/* jsx-to-string:start */}
-              <Grid gridColumns="repeat(3, 1fr)">
-                <Select
-                  label="Select"
-                  onOptionChange={() => null}
-                  options={[
-                    { value: 1, content: 'Option' },
-                    { value: 2, content: 'Option' },
-                    { value: 3, content: 'Option' },
-                    { value: 4, content: 'Option' },
-                    { value: 5, content: 'Option' },
-                    { value: 6, content: 'Option' },
-                    { value: 7, content: 'Option' },
-                    { value: 8, content: 'Option' },
-                  ]}
-                  placeholder="Default"
-                  required
-                />
-                <Select
-                  label="Select"
-                  maxHeight={150}
-                  onOptionChange={() => null}
-                  options={[
-                    { value: 1, content: 'Option' },
-                    { value: 2, content: 'Option' },
-                    { value: 3, content: 'Option' },
-                    { value: 4, content: 'Option' },
-                    { value: 5, content: 'Option' },
-                    { value: 6, content: 'Option' },
-                    { value: 7, content: 'Option' },
-                    { value: 8, content: 'Option' },
-                  ]}
-                  placeholder="Smaller"
-                  required
-                />
-                <Select
-                  label="Select"
-                  maxHeight={350}
-                  onOptionChange={() => null}
-                  options={[
-                    { value: 1, content: 'Option' },
-                    { value: 2, content: 'Option' },
-                    { value: 3, content: 'Option' },
-                    { value: 4, content: 'Option' },
-                    { value: 5, content: 'Option' },
-                    { value: 6, content: 'Option' },
-                    { value: 7, content: 'Option' },
-                    { value: 8, content: 'Option' },
-                    { value: 9, content: 'Option' },
-                    { value: 10, content: 'Option' },
-                    { value: 11, content: 'Option' },
-                    { value: 12, content: 'Option' },
-                  ]}
-                  placeholder="Larger"
-                  required
-                />
-              </Grid>
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-          <Panel header="Disabled">
-            <Text>
-              It is possible to disable the entire select component, similar to how you would disable a native HTML
-              select element, by using the <Code primary>disabled</Code> property.
-            </Text>
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          <Code primary>Select</Code> give merchants the ability to make a single selection or multiple selections from
```

Comment:
Are we missing an `s` in "give"? So it kinda reads "Select gives merchants the ability ... "

Reply:
ohh yes, good catch!